packages/manager/src/features/Kubernetes/CreateCluster/HAControlPlane.tsx

```diff
@@ -46,17 +65,31 @@ export const HAControlPlane = (props: HAControlPlaneProps) => {
         <Typography variant="inherit">HA Control Plane</Typography>
       </FormLabel>
       <HACopy />
+      {isLoadingKubernetesTypes ? (
```

Comment:
```suggestion
      {isLoadingKubernetesTypes && selectedRegionId !== '' ? (
```

The UX isn't quite right here. We'd only want to try to display the high availability price once a region has been selected.  Currently, we're seeing that loading spinner display without any region selection. `selectedRegionId` is initialized as an empty string instead of `undefined`, though I'm not sure why anymore. 

https://github.com/linode/manager/assets/114685994/ce648313-8c84-4957-8126-ed19f61b5818

It is slightly odd to have a loading spinner above the radio buttons in place of the notice while we determine whether we can get the price, but this is an edge case for slow networks or an unlikely erroring endpoint, so I think it's okay.

Reply:
Once again, I agree with you. The changes should reflect with the recent commit.